src/coreclr/inc/corinfo.h

```diff
@@ -3027,6 +3034,8 @@ class ICorStaticInfo
 
     virtual uint32_t getLoongArch64PassStructInRegisterFlags(CORINFO_CLASS_HANDLE cls) = 0;
     virtual uint32_t getRISCV64PassStructInRegisterFlags(CORINFO_CLASS_HANDLE cls) = 0;
+
+    virtual void getJitVectorInfo(CORJIT_VECTOR_INFO* jitVectorInfoPtr) = 0;
```

Comment:
nit. IMO, don't need `jit` in the name.
```suggestion
    virtual void getVectorInfo(CORJIT_VECTOR_INFO* vectorInfoPtr) = 0;
```

(looks like this "Jit" / "JIT" in the naming is somewhat viral in this PR...)

Reply:
Fixed.